Formula/age-plugin-yubikey.rb

```diff
@@ -0,0 +1,34 @@
+class AgePluginYubikey < Formula
+  desc "Plugin for encrypting files with age and PIV tokens such as YubiKeys"
+  homepage "https://github.com/str4d/age-plugin-yubikey"
+  url "https://github.com/str4d/age-plugin-yubikey/archive/v0.3.0.tar.gz"
+  sha256 "3dfd7923dcbd7b02d0bce1135ff4ba55a7860d8986d1b3b2d113d9553f439ba9"
+  license "MIT"
+  head "https://github.com/str4d/age-plugin-yubikey.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  uses_from_macos "expect" => :test
+  uses_from_macos "pcsc-lite"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    (testpath/"test.exp").write <<~EOS
+      set timeout 1
+      spawn "#{bin}/age-plugin-yubikey"
+      expect {
+        "Let's get your YubiKey set up for age!" { exit 0 }
+        timeout { exit 1 }
+      }
+    EOS
+    ENV["LANG"] = "en_US.UTF-8"
+    system "expect", "-f", "test.exp"
```

Comment:
Doesn't this work with the methods that are standard in Homebrew tests?

```suggestion
    ENV["LANG"] = "en_US.UTF-8"
    assert_match "Let's get your YubiKey set up for age!", shell_output(bin/"age-plugin-yubikey")
```

Reply:
I was using expect because age-plugin-yubikey is interactive by default, but it looks like shell_output doesn't provide standard input, so age-plugin-yubikey errors out after printing the string, which works. Done.